packages/rancher-sachet/charts/templates/configmap-pre-install.yaml

```diff
@@ -12,6 +12,17 @@ metadata:
     "helm.sh/resource-policy": keep
 data:
   config.yaml: |-
+    # please refer to the upstream documentation for configuration options:
+    #   https://github.com/messagebird/sachet/blob/master/examples/config.yaml
+    #
+    # providers:
+    #   aliyun:
+    #     region_id:
+    #     ...
+    # receivers:
+    #   - name: 'team-sms'
+    #     provider: 'aliyu'
+    #     ...
```

Comment:
I'm a little confused at the below, if they supply the values with `Values.sachet.providers`, would this default config not still be set ? I think we should only add default config if file would be empty.

Reply:
Since rancher-alerting-drivers' chart does not expose `Values.sachet.providers` which is in the sub-chart rancher-sechat, users can not configure it when installing/upgrading rancher-alerting-drivers. They will see the above comments when they edit the configMap after the app is installed.

Reply:
So the below `with` statement would only happen when used by open source users ? Technically you could hit this through the UI you would just have to manually input in the yaml editor.

Reply:
If it's not hard I think we should remove the comments if the values that fill out this file are provided.

Reply:
You are right, Rancher users can edit it as YAML to add those fields when installing the app in the Cluster Explorer.

Now the logic is that the comment will be added if neither `Values.sachet.providers` nor `Values.sachet.receivers` exits.